library/registry

```diff
@@ -1,12 +1,12 @@
-# maintainer: Joffrey F <joffrey@docker.com> (@shin-)
-# maintainer: Sam Alba <sam@dotcloud.com> (@samalba)
 # maintainer: Stephen Day <stephen.day@docker.com> (@stevvooe)
 # maintainer: Olivier Gambier <olivier.gambier@docker.com> (@dmp42)
+# maintainer: Richard Scothern <richard.scothern@gmail.com>
 
 latest: git://github.com/docker/docker-registry@0.9.1
 0.8.1: git://github.com/docker/docker-registry@0.8.1
 0.9.1: git://github.com/docker/docker-registry@0.9.1
 
-2: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
-2.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
+2: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
+2.0: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
 2.0.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
```

Comment:
@tianon I'm confused. Why require the commitish to be immutable if the file is not an immutable description of the contents of the repository? Ultimately, it sounds like these are time-based snapshots, either way, be it by git tag or a PR that changes or removes targets.